packages/core/lottie-react-native.podspec

```diff
@@ -33,7 +33,7 @@ Pod::Spec.new do |s|
     'Lottie_React_Native_Privacy' => ['ios/PrivacyInfo.xcprivacy'],
   }
 
-  s.dependency 'lottie-ios', '4.4.1'
+  s.dependency 'lottie-ios', '4.4.3'
 
   s.swift_version = '5.7'
```

Comment:
Might as well bump this to 5.9. No Xcode that can send builds to appstore connect is using 5.7 anymore, and it matches lottie ios